src/pages/about-us/culture.tsx

```diff
@@ -25,14 +84,119 @@ export default function CulturePage({
   if (!sanityCulturePage)
     throw `No Sanity document for the culture page was found.`
 
+  const icons = [
+    <ChatBubble color="primary" fontSize="large" />,
+    // @ts-ignore Color tertiary added in module augmentation but doesn't show up here.
+    <Twitter color="tertiary" fontSize="large" />,
+    <Psychology color="secondary" fontSize="large" />,
+  ]
+
   return (
     <>
+      {/* Hero Section */}
+      <Section backgroundColor="primary.main" maxWidth="md">
+        <Stack
+          spacing={3}
+          padding={3}
+        >
+          <Typography variant="h3" color="white" textAlign="center">{sanityCulturePage?.mainHeader}</Typography>
+          <PortableText color="white" textAlign="center" content={sanityCulturePage?._rawSubHeader} />
+        </Stack>
+      </Section>
+
+      {/* Blurb */}
+      <Section maxWidth="sm">
+        <Stack
+          spacing={3}
+        >
+          <Typography variant="h4" textAlign="center">{sanityCulturePage?.blurb?.title}</Typography>
+          <PortableText color="gray" textAlign="center" content={sanityCulturePage?.blurb?._rawBody} />
```

Comment:
By the way if you want it to be forced to be a certain size you can add a minHeaderSize